src/xdocs/config_coding.xml

```diff
@@ -762,6 +784,15 @@ String nullString = null;
         <source>
 &lt;module name=&quot;ExplicitInitialization&quot;/&gt;
         </source>
+
+        <p>
+          To configure the check so that it only checks for explicit null initializations:
```

Comment:
I find `explicit null initializations` weird as it doesn't reference the property name at all.
how about `for objects that explicitly initialize to null`?